app/javascript/components/subject/PauseNotifications.js

```diff
@@ -32,52 +32,79 @@ class PauseNotifications extends React.Component {
           reportError(error);
         });
     });
-  }
+  };
 
   handleSubmit = async confirmText => {
     if (await confirmDialog(confirmText)) {
       this.submit();
     }
   };
 
+  renderTooltip() {
+    const notificationStatus = this.props.patient.pause_notifications ? 'resumed' : 'paused';
+    let text = '';
+    // monitoree record is closed
+    if (!this.props.patient.monitoring) {
+      text = `Notifications cannot be ${notificationStatus} for records on the Closed line list since this monitoree is not eligible to receive notifications. If this monitoree requires monitoring, you may update this field after changing Monitoring Status to "Actively Monitoring"`;
+      // monitoree is a HH dependent
+    } else if (this.props.patient.id !== this.props.patient.responder_id) {
+      text = `Notifications cannot be ${notificationStatus} because the monitoree is within a Household, so the Head of Household will receive notifications instead. If notifications to the Head of Household should be ${notificationStatus}, you may update this field on the Head of Household record.`;
+    }
+    return (
+      <ReactTooltip id={`notifications-tooltip`} multiline={true} place="bottom" type="dark" effect="solid" className="tooltip-container mt-3">
```

Comment:
Why not use the existing `InfoTooltip` component we have ? You could dynamically figure out which info-tooltip-text you want, and pass that selector in as props, right?

Reply:
because it's a tooltip on the button, we don't want the `i icon` here I would believe